### PR TITLE
Add setup scripts

### PR DESCRIPTION
**Pull request by wtachau**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Y" | sudo apt update
```
Comment (dmrenie) [inline setup.sh:5]: `apt-get -qq` avoids the need to `echo "Y"`

Comment (dmrenie) [reply setup.sh:5]: See https://github.com/metawork/metawork/blob/main/packer/devbox/provision/system.sh#L35-L36

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  encoding: utf8mb4
+  reconnect: false
+  database: lobsters_dev
+  socket: /var/run/mysqld/mysqld.sock
+  username: metawork
+  password: password
+EOT
+
+MYSQL=$(which mysql)
+DB_SQL="CREATE DATABASE IF NOT EXISTS lobsters_dev"
```
Comment (dmrenie) [inline setup.sh:23]: Can we use `rails db:create` instead of these adhoc SQL commands?

Comment (wtachau) [reply setup.sh]: i spent some time trying to get it to work, but there were some permissions errors with setting up the user. doesn't seem like it is worth sinking more time into it

**Pull request merged**